logic/common/utils_file.py

```diff
@@ -0,0 +1,191 @@
+'''
+This file contains:
+ - Custom printing funcitons
+ - Reader function for choose the root from TOML
+
+'''
+
+#==================================================#
+'''Import & Default Constants'''
+
+import time
+import os
+import toml
+
+DEFAULT_ROOT_TOML = "input/root_dir.toml"
+
+
+
+#==================================================#
+'''Print Functions'''
+
+def print_linebreak(upper_spacing = 0, lower_spacing = 0, linebreak_char = '='):
+	'''Prints a divider, can add extra spacing above and/or below'''
+	for spacing_count in range(upper_spacing):
+		print("")
+	print(linebreak_char*50)
+	for spacing_count in range(lower_spacing):
+		print("")
+
+def _print(print_str):
+	'''Alternate print function'''
+	print(f"\t{print_str}")
+
+def _print_list(list_s):
+	'''Print all the contents of a list of string'''
+	for line in list_s:
+		print(line)
+
+
+
+#==================================================#
+'''File I/O'''
+
+def check_file_exist(filepath):
+	'''Return False if file is not found, may remove since it's just one-liner'''
```

Comment:
Yeah, we don't need this function. The wrap around doesn't add any extra clarity, since
os.path.exists(filepath) is super clear already